src/averageAges.js

```diff
@@ -37,7 +42,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const isMother = people.filter(person => person.mother)
```

Comment:
it would be better to use another variable namings
```suggestion
  const motherNames = people.filter(person => person.mother)
```